bin/wasm-node/javascript/src/index-browser.ts

```diff
@@ -121,23 +131,213 @@ function trustedBase64Decode(base64: string): Uint8Array {
           config.onMessage(new Uint8Array(msg.data as ArrayBuffer));
       };
 
-  } else {
-      throw new ConnectionError('Unrecognized multiaddr format');
-  }
+      return {
+        close: (): void => {
+            connection.onopen = null;
+            connection.onclose = null;
+            connection.onmessage = null;
+            connection.onerror = null;
+            connection.close();
+        },
 
-  return {
-    close: (): void => {
-        connection.onopen = null;
-        connection.onclose = null;
-        connection.onmessage = null;
-        connection.onerror = null;
-        connection.close();
-    },
+        send: (data: Uint8Array): void => {
+            connection.send(data);
+        },
 
-    send: (data: Uint8Array): void => {
-        connection.send(data);
-    },
+        openOutSubstream: () => { throw new Error('Wrong connection type') }
+      };
+  } else if (webRTCParsed != null) {
+    let pc: RTCPeerConnection;
+
+    const targetPort = webRTCParsed[3];
+    if (forbidWebRTC || targetPort == '0') {
+        throw new ConnectionError('Connection type not allowed');
+    }
+
+    // Create a new peer connection.
+    pc = new RTCPeerConnection();
+
+    pc.onconnectionstatechange = (_event) => {
+      console.log(`conn state: ${pc.connectionState}`);
+      if (pc.connectionState == "closed") {
+          config.onConnectionClose("");
+      }
+    };
+
+    // Create an initial data channel with a predefined `id = 1`, which is used
+    // to verify remote peer's identity.
+    const dataChannel = pc.createDataChannel("data", { id: 1, negotiated: true });
+
+    pc.onnegotiationneeded = async (_event) => {
+        // Create a new offer and set it as local description.
+        const sdpOffer = (await pc.createOffer()).sdp!;
+        await pc.setLocalDescription({ type: 'offer', sdp: sdpOffer });
+
+        console.log("LOCAL OFFER: " + pc.localDescription!.sdp);
+
+        const ipVersion = webRTCParsed[1] == 'ip4'? '4' : '6';
+        const targetIp = webRTCParsed[2];
+        const ufrag = (webRTCParsed[5] || '');
+
+        // Transform ufrag (multibase-encoded multihash) to fingerprint (upper-hex;
+        // each byte separated by ":").
+        //
+        // To add additional decoders use `or`: `base64url.decoder.or(base32.decoder)`.
+        const fingerprint = new Uint8Array(base64url.decode(ufrag)).join(':').toUpperCase();
```

Comment:
Since it's a multibase, the base64-encoded multihash-encoded certificate hash starts with a letter (I suppose 'm', 'M', 'u', or 'U').
See https://github.com/multiformats/multibase#multibase-table

Also, once base64-decoded, you still need to remove one byte for the sha-256 multihash prefix (should be 0x12) and one byte for the size (0x20).
See https://github.com/multiformats/multihash#format